scripts/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ if(IFACE STREQUAL "host")
     if(WIN32)
         list(APPEND ONEDAL_LIBRARIES "onedal_dll")
     else()
-        list(APPEND ONEDAL_LIBRARIES "onedal")
+        set(ONEDAL_LIBRARIES "onedal" "onedal_core" "onedal_thread")

```

Comment:
Please, make it consistent for Windows:
I suggest to remove lines 29-35, and place the same lines in Windows branches (80, 91).
For example,
`set(ONEDAL_LIBRARIES "onedal_dll", "onedal_core_dll", "onedal_thread_dll")`